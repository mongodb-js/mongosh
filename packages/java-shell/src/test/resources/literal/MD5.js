// command checkResultClass
new MD5('31323334')
// command checkResultClass
MD5('31323334')