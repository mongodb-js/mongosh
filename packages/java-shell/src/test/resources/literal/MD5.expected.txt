BinaryResult: { "$binary" : "MTIzNA==" , "$type" : 5}
BinaryResult: { "$binary" : "MTIzNA==" , "$type" : 5}